tasks/install_shell.sh

```diff
@@ -407,8 +407,9 @@ install_file() {
       ;;
     "deb")
       info "installing puppetlabs apt repo with dpkg..."
-      dpkg -i "$2"
+      dpkg -i --force-conflicts "$2"
       apt-get update -y
+      apt-get install --fix-broken -y
```

Comment:
What is `--fix-broken` addressing?

Reply:
I can update with dpkg using the `--force-conflicts` flag. However when `apt-get` is used to install this results in a conflict between puppet5 and 6. 
```
The following packages have unmet dependencies:
 puppet6-release : Conflicts: puppet5-release but 5.0.0-2bionic is to be installed
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).
```
The error message indicates running the `--fix-broken` install without listing packages. I am looking in to how to `specify a solution`.